### ISSUE TITLE
[accessibility][voice reader] Mac Voice Reader reads incorrect ShimmeredDetailsList row count while using Sticky component 

### ISSUE THREAD

**Issue by lutorre** [OfficeDev/office-ui-fabric-react#10682]
### Environment Information

- **Package version(s)**: MacOS Mojave 10.14.5
- **Browser and OS versions**: Chrome, Safari

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

 Mac Voice Reader reads incorrect ShimmeredDetailsList row count while using Sticky component. When using a ShimmeredDetailsList and applying a Sticky components to the headers voice over doesn't take into account the first row (headers). 

### Please provide a reproduction of the issue in a codepen:

1- Create a ShimmeredDetailsList with 5 rows (including headers)
2- Wrap the headers within a Sticky component in onRenderDetailsHeader
3- Turn on Voice Over and select the list using Tab key

example:

private onRenderDetailsHeader = (
        props: IDetailsHeaderProps,
        defaultRender: (props?: IDetailsHeaderProps) => JSX.Element
    ): JSX.Element => {
        return (
            <Sticky
                stickyPosition={StickyPositionType.Header}
                isScrollSynced={true}
            >
                {defaultRender(props)}
            </Sticky>
        );
    };

#### Actual behavior:

After selecting the list, the screen readers says: "Table, 4 rows"

#### Expected behavior:

After selecting the list, the screen readers says: "Table, 5 rows"

**Comment by khmakoto** [comment]
Hi @lutorre, thanks for filing this issue! I'll dig into it to see where the problem is coming from.

**Comment by khmakoto** [comment]
Hi @lutorre, I've made [this codepen](https://codepen.io/khmakoto/pen/qBBdbKj?editors=0010) following the instructions you posted above and checked what the behavior is with Edge and Narrator. After trying this, I can't seem to repro the bug you're having with VoiceOver on the Mac. Can you take a look at the codepen, check if it correctly reproes your scenario for you and if that's not the case, then modify the codepen so it reproes your scenario?

Alternatively, if the codepen I've provided does repro your scenario, then, since our screen reader support is limited to Edge + Narrator and it seems that we're using the correct aria values here, my recommendation would be to provide feedback to VoiceOver.
